package.json

```diff
@@ -13,10 +13,12 @@
     "nan": "^2.17.0"
   },
   "devDependencies": {
-    "tree-sitter-c": "git://github.com/tree-sitter/tree-sitter-c.git",
-    "tree-sitter-cli": "^0.20.7"
+    "tree-sitter-c": "^0.20.6",
+    "tree-sitter-cli": "^0.20.8"
```

Comment:
I see, `npm update` doesn't update devDependencies. Can this be automated in the update deps workflow?